README.md

```diff
@@ -70,17 +70,16 @@ While inside a matrix, array, align, or cases environment,
 
 
 ### Conceal
-*This feature must be enabled in the plugin settings.*
+**This feature must be enabled in the plugin settings!**
```

Comment:
Can we keep the "." here? I think the "!" feels less professional.

Reply:
Thank you for the feedback. Got it.